plugins/woocommerce-blocks/assets/js/blocks/mini-cart/block.tsx

```diff
@@ -76,24 +76,6 @@ const MiniCartBlock = ( attributes: Props ): JSX.Element => {
 		}
 	}, [ cartIsLoading, cartIsLoadingForTheFirstTime ] );
 
-	useEffect( () => {
```

Comment:
I removed this affect because all local storage/caching has moved to data store level.